docs/integrations/scheduled-commands.md

```diff
@@ -0,0 +1,60 @@
+---
+id: scheduled-commands
+title: Scheduled Commands
+---
+
+### available from XSOAR version 6.2.0 and up
+
+<img width="533" src="../doc_imgs/integrations/polling-command.png"></img>
+
+It's possible for a command to schedule a future execution for another command with predetermined `next_run` time and `args` as well as `timeout`.
+
+Use cases for using a scheduled commands include:
+1. ***Polling Flow*** - The command cannot return the full result in a single execution (likely because it's waiting for a remote process to finish execution). Scheduled commands enable to set the command to try again later, and return the full result when it can. example use cases: `Sandbox Detonation`, `Autofocus samples search`.
```

Comment:
```suggestion
1. ***Polling Flow*** - The command cannot return the full result in a single execution (likely because it's waiting for a remote process to finish execution). Scheduled commands enable to set the command to try again later, and return the full result when it can. Example use cases are `Sandbox Detonation` and `Autofocus samples search`.
```